.releaserc

```diff
@@ -14,9 +13,11 @@
             "path": "build/markdown/spec.md",
             "label": "Github Markdown distribution"
           }
-        ]
+        ],
+        "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}\n\nThis commit has been made automagically via .releaserc"
```

Comment:
```suggestion
        "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}\n\nCommit generated by .releaserc"
```

Reply:
just trying to get it more inline with this ticket:
https://github.com/open-rpc/generator-client/issues/42